### DISCUSSION TITLE
SQL查询是否可以像SQL上线那样根据资源组来过滤？

### DISCUSSION THREAD

**Discussion by jy00278439** [hhyo/Archery#1847] [Ideas]
SQL查询是否可以像SQL上线那样根据资源组来过滤？

**Comment by carlors** [comment]
可以配置专门的从库给SQL查询功能用，从库像SQL上线一样配置和主库一样的资源组，然后实例标签那里，选择“支持查询”，不要选择“支持上线”即可。
